CMSgov/design-system
# [WNMGDS-581] Reorganize doc site scss

### Summary
Child design system doc sites need to import styles from `design-system-docs/src/index.scss` and `design-system-docs/src/example.scss`. This make it easier to do so while using the child design system styles. By separating out 2 extra scss files, we can provide scss entry files **without** imports to the core CMSDS.

### How to test
- Ensure no doc site visual changes
- Go to any of the child design systems, replace `docs/src/index.scss` and `docs/src/example.scss` with new imports to `design-system-docs/src/styles/index.scss` and `design-system-docs/src/styles/example.scss`. It should look like:
```
// Override default assets path
$font-path: './fonts';
$image-path: './images';

// Use overriden css from child DS src
@import '../../src/styles/index';

@import '@cmsgov/design-system-docs/src/styles/example';
// or @import '@cmsgov/design-system-docs/src/styles/index';
```
